easyapplybot.py

```diff
@@ -41,23 +56,22 @@ def __init__(self,username,password, language, positions, locations, resumeloctn
         self.options = self.browser_options()
         self.browser = driver
         self.wait = WebDriverWait(self.browser, 30)
-        self.start_linkedin(username,password)
-
+        self.start_linkedin(username, password)
 
     def browser_options(self):
         options = Options()
         options.add_argument("--start-maximized")
         options.add_argument("--ignore-certificate-errors")
-        #options.add_argument("user-agent=Chrome/51.0.2704.79 Safari/537.36 Edge/14.14393")
-        #options.add_argument('--headless')
+        # options.add_argument("user-agent=Chrome/51.0.2704.79 Safari/537.36 Edge/14.14393")
+        # options.add_argument('--headless')
         options.add_argument('--no-sandbox')
-        #options.add_argument('--disable-gpu')
-        #options.add_argument('disable-infobars')
+        # options.add_argument('--disable-gpu')
+        # options.add_argument('disable-infobars')
         options.add_argument("--disable-extensions")
```

Comment:
# Disable webdriver flags or you will be easily detectable
        options.add_argument("--disable-blink-features")
        options.add_argument("--disable-blink-features=AutomationControlled")